runtime/vm/ContinuationHelpers.cpp

```diff
@@ -401,6 +401,32 @@ freeContinuation(J9VMThread *currentThread, j9object_t continuationObject, BOOLE
 #if JAVA_SPEC_VERSION >= 24
 		/* Remove reverse link to vthread object. */
 		continuation->vthread = NULL;
+
+		if (NULL != continuation->objectWaitMonitor) {
```

Comment:
This code block is also used in `enterContinuation`. An helper should be added for it to reduce duplication.

Reply:
updated